src/Compilers/Core/Portable/MemberDescriptor.cs

```diff
@@ -133,8 +136,14 @@ private static short ReadTypeId(Stream stream)
             }
         }
 
-        private static void ParseMethodOrPropertySignature(ImmutableArray<byte>.Builder builder, Stream stream)
+        private static void ParseMethodOrPropertySignature(ImmutableArray<byte>.Builder builder, Stream stream, int arity)
         {
+            // Constraints
+            for (int i = 0; i < arity; i++)
+            {
+                ParseConstraints(builder, stream);
```

Comment:
>ParseConstraints(builder, stream); [](start = 16, length = 34)

Consider keeping constraints as a separate entry in the MemberDescriptor, constraints are not really part of a signature.